### DISCUSSION TITLE
Not working for javascript on vscode

### DISCUSSION THREAD

**Discussion by rthangam** [community/community#6979] [Copilot Conversations]
I got an invite to try copilot so i followed the steps mentioned to enable it on VS Code, but still i find i doesn't suggest any code ?

**Comment by isolcat** [comment]
In fact, I didn't encounter this situation. Maybe you can try to reload copilot?

**Reply by rthangam** [reply]
Tried to reload, uninstall and install etc but nothing works. I am using Macbook.

**Reply by isolcat** [reply]
I'm sorry about this. I can't help you, because I'm using Windows 11.

**Reply by rthangam** [reply]
Thanks @isolcat for taking your time to check this issue.

**Reply by rthangam** [reply]
Also tried View -> Output and selected Github Copilot from drop down for output section but still it doesn't show any output there as well.

**Reply by efreibe** [reply]
If you try Help -> Toggle Developer Tools, is there something suspicious in the console that could indicate any failure when loading the extension?

**Reply by rthangam** [reply]
Tried but no issues there. After installing the extension i could see copilot extension on the bottom panel.

**Reply by DevPaw** [reply]
ladies & gentlemen......We Gotem

for me it worked and I am relieved
1) do what everyone is saying, installing, reopening
2) click the bell icon on bottom right there click agree 
![image](https://user-images.githubusercontent.com/83178531/139459235-c750b44f-0508-473e-bb2e-40f5ae9ecfa6.png)

3) that's it now try

found this answer on Stackoverflow - https://stackoverflow.com/questions/68253302/github-copilot-commands-not-working-and-showing-error/69067153#69067153?newreg=3dda325512514997af78535813f46e3f

**Reply by nationsahil** [reply]
hey thanks for this i was stucked on this op thanks for asking also ,, and yeah copilot is amazing

**Reply by rthangam** [reply]
Still not working for me tried login/logout from github and even tried unistall/install copilot extension. Is there any extension which is causing the issue for copilot ?

**Comment by rthangam** [comment]
It is working now :)

**Reply by DevPaw** [reply]
Good, enjoy it now, Github Copilot is soooo good. It like reads your mind and gives you suggestions
